test/src/test/testtool/RestListener/scenario01.properties

```diff
@@ -13,7 +13,7 @@ step1.dir.init.write = 01/dummy.txt
 step2.java.XmlQuerySender.write = 01/xqs-init-in.xml
 step3.java.XmlQuerySender.read = 01/xqs-init-out.xml
 step4.java.RestSender_ArchiveDoc.write = 01/archive-in.xml
-step5.java.RestSender_ArchiveDoc.read = 01/archive-out.xml
+step5.java.RestSender_ArchiveDoc.read = 01/archive-out-check.xml
 step6.java.XmlQuerySender.write = 01/xqs-select-in.xml
 step7.java.XmlQuerySender.read = 01/xqs-select-out.xml
 step8.java.RestSender_GetDoc.write = 01/archive-out.xml
```

Comment:
Ik zou het mooier vinden als je de `archive-out-check.xml` terugnoemt naar `archive-out.xml` en de `step8.java.RestSender_GetDoc.write` als value `get-in.xml` noemt oid. dan is het duidelijker welke in's en out's bij elkaar horen.